gpii/node_modules/flowManager/src/PCPChannel.js

```diff
@@ -45,7 +45,7 @@ fluid.defaults("gpii.pcpChannel.sessionBinder", {
 /**
  * The PCP channel maintains its own model of the PCP state, it contains the following data:
```

Comment:
Given you are renaming the entire world, could you also globally rename PCP -> PSP, pcp -> psp, thanks!